VaticanBAV/et/BAVet20.xml

```diff
@@ -245,14 +252,17 @@ schematypens="http://relaxng.org/ns/structure/1.0"
           
                <history>
                   <origin>
-                     <origPlace>
-                        <placeName ref="wd:Q220"/>
-                        
-                     </origPlace>
-                     <origDate notBefore="1400" notAfter="1499"/>
-                     <note/>
-                     <p/>
+                     <origDate notBefore="1400" notAfter="1487"/>
                   </origin>
+                  <provenance>Origin and provenance of the manuscript are unknown.
+                     <bibl><ptr target="bm:Krebs2021Kingship"/><citedRange unit="page">138–139</citedRange></bibl>
+                     speculates that it might have been brought from <placeName ref="LOC3010Ethiop">Ethiopia</placeName>
+                     to <placeName ref="LOC3957Jerusa">Jerusalem</placeName> by <persName ref="PRS14337Baptista"/>
+                  after <date notBefore="1484" notAfter="1485">1484–1485</date>.</provenance>
+                  <acquisition notAfter="1487">The manuscript was already part of the library’s collection by <date>1487</date>,
```

Comment:
You mean the library in the Vatican and not in Jerusalem. Otherwise one might think, that Brocchi borrowed it from the Jerusalem monastic library.